temporalcli/commandsmd/commands.md

```diff
@@ -1,487 +1,1185 @@
+Remove without confirmation:
+
+```
+temporal operator search-attribute remove \
+    --name YourAttributeName \
+    --yes false
+```
 
 #### Options
 
-* `--name` (string[]) - Search Attribute name. Required.
-* `--yes`, `-y` (bool) - Confirm prompt to perform deletion.
+* `--name` (string[]) -
+  Search Attribute name.
+  Required.
+* `--yes`, `-y` (bool) -
+  Override confirmation request before removal.
+
+### temporal schedule: Perform operations on Schedules
 
-### temporal schedule: Perform operations on Schedules.
+Create, use, and update Schedules that allow Workflow Executions to
+be created at specified times:
 
-Schedule commands allow the user to create, use, and update Schedules.
-Schedules allow starting Workflow Execution at regular times.
+```
+temporal schedule [commands] [options]
+```
+
+For example:
+
+```
+temporal schedule describe \
+    --schedule-id "YourScheduleId"
+```
 
 #### Options
 
 Includes options set for [client](#options-set-for-client).
 
-### temporal schedule backfill: Backfills a past time range of actions.
+### temporal schedule backfill: Backfill past actions
 
- The `temporal schedule backfill` command runs the Actions that would have been run in a given time
-interval, all at once.
 
- You can use backfill to fill in Workflow Runs from a time period when the Schedule was paused, from
-before the Schedule was created, from the future, or to re-process an interval that was processed.
+Batch-execute actions that would have run during a specified time interval.
+Use this command to fill in Workflow Runs from when a Schedule was paused,
+from before a Schedule was created, from the future, or to re-process an
+interval that was previously executed.
 
-Schedule backfills require a Schedule ID, along with the time in which to run the Schedule. You can
-optionally override the overlap policy. It usually only makes sense to run backfills with either
-`BufferAll` or `AllowAll` (other policies will only let one or two runs actually happen).
+Backfills require a Schedule ID and the time period covered by the request. 
+It's best to use the `BufferAll` or `AllowAll` policies.
 
-Example:
+For example:
 
 ```
-  temporal schedule backfill           \
-    --schedule-id 'your-schedule-id'   \
-    --overlap-policy BufferAll         \
-    --start-time 2022-05-01T00:00:00Z  \
-    --end-time   2022-05-31T23:59:59Z
+  temporal schedule backfill \
+    --schedule-id "YourScheduleId" \
+    --start-time "2022-05-01T00:00:00Z" \
+    --end-time "2022-05-31T23:59:59Z" \
+    --overlap-policy BufferAll
 ```
 
 #### Options set for overlap policy:
 
-* `--overlap-policy` (string-enum) - Overlap policy. Options: Skip, BufferOne, BufferAll, CancelOther, TerminateOther, AllowAll. Default: Skip.
+* `--overlap-policy` (string-enum) -
+  Overlap policy.
+  Options: Skip, BufferOne, BufferAll, CancelOther, TerminateOther, AllowAll.
+  Default: Skip.
```

Comment:
We should probably try to describe each of the policies in the long description above. It's kinda-sorta clear what these do from their names, but having more concrete descriptions for each of them would be good because it helps the reader to decide which applies in their particular situation.

Reply:
Adding

Reply:
```
The policies include:

* AllowAll: Allow unlimited concurrent Workflow Executions. This
  significantly speeds up the backfilling process on systems that
  support concurrency. Make sure running Workflow Executions will not
  interfere with each other.
* BufferAll: Buffer all incoming Workflow Executions while waiting 
  for the running Workflow Execution to complete.
* Skip: If a previous Workflow Execution is still running, discard
  new Workflow Executions.
* BufferOne: Same as 'Skip' but buffer a single Workflow Execution
  to be run after the previous Execution completes. Discard other
  Workflow Executions.
* CancelOther: Cancel the running Workflow Execution and replace it
  with the incoming new Workflow Execution.
* TerminateOther: Terminate the running Workflow Execution and replace it
  with the incoming new Workflow Execution.
```